### ISSUE TITLE
Groups added with bridge/config/add_group are missing the 'retain' property

### ISSUE THREAD

**Issue by sjorge** [Koenkk/zigbee2mqtt#2264]
# Bug Report
## What happened
Added a test group by publishing
`test` to `zigbee2mqtt/bridge/config/add_group`, this resulted in

```
groups:
  '1':
    friendly_name: 'test'
```

To be added to the configuration (so far so good), however on restart zigbee2mqtt fails to start with the following error:

```
zigbee2mqtt:error 2019-11-03T19:38:49: Refusing to start, configuration.yaml is not valid, found the following errors:
zigbee2mqtt:error 2019-11-03T19:38:49:   - groups['1'] should have required property 'retain
```

This makes sense as 'retain' seems to be a required property for groups.

## What did you expect to happen
The group to have retain set to either 0 or 1

## How to reproduce it (minimal and precise)
Add a new group via mqtt

## Debug Info
zigbee2mqtt version: 1.6.0 @ dev from 20191103
CC253X firmware version: n/a

**Comment by sjorge** [comment]
I'll see if I can fix this while I am adding support for creating a group with an id set.

**Comment by sjorge** [comment]
I have a fix for this, will open PR for feedback in a bit...

**Comment by sjorge** [comment]
Now fixed after PR #2266 got merged
